stablehlo/transforms/VhloToVersion.cpp

```diff
@@ -77,18 +78,87 @@ FailureOr<Version> validateTargetVersion(llvm::StringRef versionRef,
   return targetVersion;
 }
 
+template <typename AttrOrType>
+bool isDialect(AttrOrType at, llvm::StringRef dialectName) {
```

Comment:
`isFromDialect`? During the previous review, the name looked off to me, but I couldn't come up with a better idea. Now I think I have.

Reply:
This doesn't look addressed. Forgot to push changes?